MdePkg/Library/StackCheckLibNull/StackCheckLibNull.inf

```diff
@@ -0,0 +1,33 @@
+## @file
+#  Null library instance for StackCheckLib which can be included
+#  when a build needs to include stack check functions but does
+#  not want to generate stack check failures.
+#
+#  Copyright (c) Microsoft Corporation.
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005

```

Comment:
Please use the right value here (1.29 if I am not mistaken). Also no need to use the hex representation.